moveit_planners/ompl/ompl_interface/src/detail/constrained_goal_sampler.cpp

```diff
@@ -41,6 +41,8 @@
 
 #include <utility>
 
+static const std::string LOGNAME{ "constrained_goal_sampler" };
```

Comment:
Please put this inside the ompl_interface namespace (the same applies to all cases below). In addition, write it as
```
constexpr char LOGNAME[] = "constrained_goal_sampler";
```
Constexprs offer the compilers more options for optimization. A std::string cannot be created via constexpr.

Reply:
Ok!

Can I put all class method definitions inside the namespace (as is done in many other parts of MoveIt, but not [here](https://github.com/ros-planning/moveit/blob/877e1d51560f701b60b79e04a852ec104ad77de0/moveit_planners/ompl/ompl_interface/src/planning_context_manager.cpp#L88) for example). Or is the preferred way to do `ompl_interface::LOGNAME` in all the log statements?